packages/tasks/src/model-libraries-downloads.ts

```diff
@@ -0,0 +1,20 @@
+/**
+ * This file contains the (simplified) types used
+ * to represent queries that are made to Elastic
+ * in order to count number of model downloads
+ *
+ * Read this doc about download stats on the Hub:
+ *
+ * https://huggingface.co/docs/hub/models-download-stats
+ *
+ * see also:
+ * https://www.elastic.co/guide/en/elasticsearch/reference/current/query-dsl-bool-query.html
+ */
+
+export type ElasticBoolQueryFilter =
+	// match a single filename
+	| { term?: { path: string } }
+	// match multiple possible filenames
+	| { terms?: { path: string[] } }
+	// match a wildcard
+	| { wildcard?: { path: string } };
```

Comment:
note to reviewers: I only expose the `filter` option because:
- it's simpler: every library except for diffusers is using it.
- i had typings issues/conflicts with the Chai library when defining a type that contains a property named `should`. cc @coyotte508. Rather than fightining the tooling, i decided to work around it